docs/quickstart/cross-chain-transfers.md

```diff
@@ -169,13 +169,17 @@ which returns:
   unsignedTx: {
     networkID: 5,
     blockchainID: '2JVSBoinj9C2J33VntvzYtVJNZdN2NKiwwKjcumHUWEb5DbBrm',
-    outputs: [],
-    inputs: [ [Object] ],
+    outputs: [ [Object] ],
+    inputs: [ [Object], [Object] ],
     memo: '0x41564d207574696c697479206d6574686f64206275696c644578706f7274547820746f206578706f7274204156415820746f2074686520432d436861696e2066726f6d2074686520582d436861696e',
     destinationChain: 'yH8D7ThNJkxmtkuv2jgBa4P1Rn3Qpr4pPr7QYNfcdoS6k6HWp',
     exportedOutputs: [ [Object] ]
   },
   credentials: [
+    {
+      fxID: 'spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ',
+      credential: [Object]
+    },
```

Comment:
are there two elements with the same value under `credentials`?

Reply:
confirmed, yes there is